less/common/Modal.less

```diff
@@ -107,37 +107,20 @@
   }
 
   .off.Checkbox--switch .Checkbox-display {
-    background: @muted-more-color;
+    background: var(--muted-more-color);
   }
 }
 .Modal-footer {
   border: 0;
   padding: 20px;
   text-align: center;
-  color: @muted-color;
-}
-.Modal-loading {
```

Comment:
Dead code, right? Not finding any references to it, but still wanted to make sure.

Reply:
Yup, dead code, I tracked it to the commit where the JSX was removed, I've seen this happen quiet a few times where the relevant UI is removed but the CSS is forgotten, we should try to watch out for that ourselves too.